webportal/lambda/.chalice/config.json

```diff
@@ -86,8 +86,8 @@
         "TABLENAME_USER_STACKS": "prod-UserStacksTable",
         "TABLENAME_AVAILABLE_DATASET": "prod-AvailableDataset",
         "TABLENAME_AUTOEXPORT_USERS": "prod-AutoExportUsersTable",
-        "RESTAPIID": "cyfca70occ",
-        "AUTHORIZERID": "r6ck6c",
+        "RESTAPIID": "bzzlrrsxa4",
```

Comment:
should these values be hard coded?

Reply:
Yes. Otherwise it would take extra API calls (and latency) for all API requests.